src/ZEO/ClientStorage.py

```diff
@@ -99,6 +99,7 @@ def __init__(self, addr, storage='1', cache_size=20 * MB,
                  wait=True,
                  drop_cache_rather_verify=True,
                  username=None, password=None, realm=None,
```

Comment:
If you have username and password in credentials, why do you also need them in keyword arguments?